python/cudf/utils/utils.py

```diff
@@ -265,6 +265,64 @@ def get_dummies(df, prefix='', prefix_sep='_', cats={}, columns=None,
     ], axis=1)
 
 
+def get_dummies_v2(df, prefix='', prefix_sep='_', dummy_na=False, columns=None,
```

Comment:
Why do we need a second get_dummies function?

Reply:
Added a second get_dummies because of introduction of 3 new non-functional function parameters(`dummy_na`, `sparse`, `drop_first`). Pandas get_dummies API has these 3 function parameters extra. I could add these extra params in the existing API too but would change the order of params a bit, if that is okay I can make those addition to existing function itself.

Reply:
In these situations, we would want to include them in the API but require them to be the default value. In general, if we are missing function parameters we will likely run into issues with dask. Simply allowing them to be passed (even if we don't currently use them) is enough.

For example:

https://github.com/rapidsai/cudf/blob/f742ce8579f9fbf081c595f1d568aa21a95ecc93/python/cudf/dataframe/series.py#L1562

We're not actually doing anything with `skipna`, but it's necessary.

Reply:
@beckernick made suggested changes.